### PR TITLE
Table 11 Added

### PR DESCRIPTION
**Pull request by UtkarshTiwari1750**

### CODE REVIEW

File: modules/tables/table_11_Utkarsh.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<style>
+    *{
```
Comment (NtemKenyor) [inline modules/tables/table_11_Utkarsh.html]: using a selector of "*" would change the style for some content that are outside your module.

File: modules/tables/table_11_Utkarsh.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        overflow: hidden;
+    }
+    table{
+        background: white;
+        width: 60%;
+        height: 40%;
+    }
+    tr{
+        text-align: center;
+        /* all:unset; */
```
Comment (NtemKenyor) [inline modules/tables/table_11_Utkarsh.html]: Even though it's not mandatory, You could add comments to explain the next steps.

**Review by NtemKenyor — CHANGES_REQUESTED** [review]
Everything looks except the thing pointed out. You module would be imported into different projects and should not style elements outside it's content(itself).

This is a great contribution, Make the requested change (s) and we would be here to accept. Than you.

**Comment by UtkarshTiwari1750** [comment]
I have Updated Please Check

**Pull request closed without merging**